---
permalink: /James_Sacco_resume.pdf/
title: "Resume"
author_profile: true

---

**JAMES SACCO![](Aspose.Words.eb3063da-8c09-487f-9519-44c07f4ce427.001.png)**

**Bioinformatics Scientist**

**Department of Gene and Cell Therapy** **ASC Therapeutics** 

**Milpitas, CA, 95035**

Independent, self-motivated bioinformatics scientist, with five years of experience in cancer immunotherapy and gene therapy for rare diseases. Specialist in computational biology of CRISPR gene editing. An analytical thinker and quick learner, with experience in next-generation sequencing (NGS) methods, especially RNA-Seq, and construction of reproducible and robust analysis pipelines. Integrated genotype phenotype data to predict disease severity. Developed machine learning applications to analyze population-scale genomic and real-world patient data. Experienced with cross-functional teams of diverse

cultural and technical backgrounds.

CONTACT INFO

√† <jsacco001@gmail.com>

√Å [jsacco1.github.io](https://jsacco1.github.io/)

- [github.com/jsacco1](https://github.com/jsacco1)

√° [linkedin.com/in/james-sacco/](https://www.linkedin.com/in/james-sacco/)

¬ï +1 786-473-9993 For more information, please contact me via email.

¬± EMPLOYMENT

Present **Bioinformatics Scientist**

| ASC Therapeutics ÔèÖ Milpitas, CA SKILLS

2021 ‚óè Support NGS experiments to quantify and characterize on/off-targets of

gene modifications by performing data wrangling and analysis, using Extensive experience with NGS open-source methods (e.g., CALITAS, CHANGE-Seq, CRISPResso2,

data analysis and workflow GUIDE-Seq, Cas-OFFinder) and custom scripts.

development

- Query, retrieve, and integrate data from public genomics databases, to

characterize alignments between CRISPR target sequences and guide Hands-on experience in

RNAs.

sequencing pipelines and tools,

- Compare concordance and sensitivity of five CRISPR off-target such as: Illumina, BEDtools, detection methods. SAMtools, DESeq, edgeR
- Spearhead implementation of reproducible, robust bioinformatics data

practices Proficient in several *in silico* CRISPR gene editing and RNA-

2020 **Biomarker Data Analyst** Seq analysis tools and methods

| Genentech ÔèÖ South San Francisco, CA

2019 ‚óè Established department-first machine learning pipeline to study effects **Programming Skills**

of biomarker operations on quality of cancer immunotherapy assays.

- Generated over ten percent annual operation cost reduction via analysis Ôì∑ R

of vendor performance over twenty clinical sites.

Ôè¢ Python

- Upgraded data operations for six data streams, including flow cytometry,

biomarker operations, and oncology data warehouses.

ÔÖº Linux

- Monitored and solved sample data quality issues for two clinical trial

arms. ÔÑ¶ Git

- GitHub

2019 **Data Curator![](Aspose.Words.eb3063da-8c09-487f-9519-44c07f4ce427.002.png)**

| Genentech ÔèÖ South San Francisco, CA 2018 ‚óè Collaborated with AI engineers to create an ETL pipeline for multi-modal

survival prediction and patient stratification, by using Python machine

learning and R/Bioconductor to integrate gene panel and RNA-Seq data.

- Tested and refined a textual entity and geolocation matching pipeline for clinical trial site management, resulting in over 90% accuracy.
- Designed ETL data pipeline for integration of real-world EHRs into deep learning module, using Python, SQL, and Apache Spark.
- Organized and managed close coordination of gRED Artificial Intelligence, data management, and DevOps functions, to bring clinical machine learning from prototype to production.

2018 **Data Curator, Bioinformatics Analyst**

| BioMarin Pharmaceutical ÔèÖ San Rafael, CA 2017 ‚óè Established an integrated genetic and curated literature workflow to

predict incidence and prevalence rates of over sixty rare, genetic disorders, with R statistical and visualization packages (ggplot2) and MATLAB.

- Upgraded and tested RNA-Seq data pipeline (STAR2, SAMtools, Bioconductor) to discern disease contribution of rare variants to neural disease.
- Spearheaded development of three relational databases derived from Hail HPC population genomics platform, in collaboration with software engineer.
- Co-authored manuscript submitted to peer-reviewed publication, on prediction of disease severity in metachromatic leukodystrophy.

2017 **Research Associate**

| University of Miami Miller School of Medicine, The Miami Project to 2016 Cure Paralysis Pearse Lab

ÔèÖ Miami, FL

- Mentored research associates in developing bioinformatic skills, specifically in multiple sequence alignment and standard molecular biological protocols (restriction enzyme digestion, gel electrophoresis, spectrophotometry).
- Identified over forty putative conserved vertebrate phosphodiesterase proteins, by using multiple sequence alignment and genomic evidence.

ÔÜù EDUCATION

**Rowan University, Graduate School of Biomedical Sciences** Master of Biomedical Science ÔèÖ Stratford, NJ

**Florida International University, The Honors College**

BSc, Biological Sciences ÔèÖ Miami, FL

ÔÖõ PUBLICATIONS & POSTERS![](Aspose.Words.eb3063da-8c09-487f-9519-44c07f4ce427.003.png)

2020 **Prediction of disease severity in metachromatic**

**leukodystrophy using measures of protein activity and a**

**novel phenotype matrix.**

*American Society of Human Genetics.* **[Forthcoming on bioRxiv.org; pending acceptance in PLoS Genetics]**

M. Trinidad, X. Hong, **J. Sacco**, H.P. Nguyen, W.T. Clark, S. Froelich, J.H. LeBowitz, M.H. Gelb.

2018 **Association of HGMD and gnomAD variants of unknown**

**significance with prediction of disease incidence and**

**prevalence.**

*Annual BioMarin Research Conference.* **[Poster]**

**J. Sacco**, W.T. Clark, K. Yu, K. Wu, J.H. LeBowitz.

2016 **Regulating Axonal Responses to Injury: The Intersection**

**between Signaling Pathways Involved in Axon Myelination**

**and The Inhibition of Axon Regeneration.** *Frontiers in Molecular Neuroscience.* 2016 Jun 8;9:33.

Rao S.N., Pearse D.D. **Acknowledgement**

. LANGUAGES

**English:** Native **Spanish:** Native

ÔÄô *[Download ](https://jsacco1.github.io/James_Sacco_resume.pdf)This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown) and [**datadrivencv**](http://nickstrayer.me/datadrivencv/index.html). Code available on* ¬õ *[GitHub.*](https://github.com/jsacco1/resume)*

*Last updated on 2022-01-19.The most recent version of this resume is [available here](https://jsacco1.github.io/James_Sacco_resume.pdf).*
