---
layout: default
permalink: /cv
title: My CV
author: James Sacco
date: "2022-01-25"
params:
  pdf_mode:
    value: true
output:
  html_document:
    keep_md: true
    self_contained: true
  pagedown::html_resume:
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
knit: pagedown::chrome_print 
---

<script src="https://kit.fontawesome.com/1f72d6921a.js" crossorigin="anonymous"></script>

Aside
================================================================================

Contact Info {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i> jsacco001@gmail.com
- <i class="fas fa-link"></i> [jsacco1.github.io](https://jsacco1.github.io)
- <i class="fa fa-github"></i> [github.com/jsacco1](https://github.com/jsacco1)
- <i class="fa fa-linkedin"></i> [linkedin.com/in/james-sacco/](https://www.linkedin.com/in/james-sacco/)
- <i class="fa fa-phone"></i> +1 786-473-9993
- For more information, please contact me via email.

Skills {#skills}
--------------------------------------------------------------------------------

- Extensive experience with NGS data analysis and workflow development

-	Hands-on experience in sequencing pipelines and tools (Illumina, BEDtools, SAMtools, DESeq, edgeR, Hail)

- Proficient in several _in silico_ CRISPR gene editing and RNA-Seq analysis tools and methods

- Familiar with public genomic repositories; genomic data types, file formats, and workflows (differential gene expression, gene set enrichment, variant annotation)

- Reproducible and robust scientific programming practices

Main
================================================================================

**James Sacco  | Curriculum Vitae ** {#title}
--------------------------------------------------------------------------------

**Bioinformatics Scientist**  
**Department of Gene and Cell Therapy**  
**ASC Therapeutics**  
**Milpitas, CA, 95035**

Independent, self-motivated bioinformatics scientist, with five years of experience in cancer immunotherapy and gene therapy for rare diseases. Specialist in computational biology of CRISPR gene editing. An analytical thinker and quick learner, with experience in next-generation sequencing (NGS) methods, such as RNA-Seq, and pipeline development. Integrated genotype phenotype data to predict disease severity. Developed machine learning applications to analyze population-scale genomic and real-world patient data. Experienced with cross-functional teams of diverse cultural and technical backgrounds.

--------------------------------------------------------------------------------

Employment {data-icon=briefcase}
--------------------------------------------------------------------------------

### Bioinformatics Scientist

ASC Therapeutics

Milpitas, CA

Present - 2021

::: concise
● Quantify and characterize CRISPR gene modifications, with both custom and open-source tools for genomic data analysis.

● Query, retrieve, and integrate data from public genomics databases, to enhance on-target gene editing with sequence alignment protocols. 

● Examine concordance and sensitivity of five open-source CRISPR off-target detection methods (CALITAS, CHANGE-Seq, CRISPResso, GUIDE-Seq, and Cas-OFFinder).

● Streamline high-performace analyses for bench scientists by constructing custom data pipelines.

● Spearhead implementation of version control, and programming best practices.
:::

### Biomarker Data Analyst II
Genentech (Contract)

South San Francisco, CA

2020 - 2019

::: concise
● Established department-first machine learning pipeline to study effects of biomarker operations on quality of cancer immunotherapy assays.

●	Upgraded data operations for six data streams, including flow cytometry, biomarker operations, and oncology data warehouses.

●	Monitored and solved sample data quality issues for two cancer immunotherapy trial arms.
:::

### Data Curator

Genentech (Contract)

South San Francisco, CA

2019 - 2018 

::: concise
●	Collaborated with AI engineers to create an ETL pipeline for multi-modal survival prediction and patient stratification, by using Python machine learning and R/Bioconductor to integrate gene panel and RNA-Seq data.

●	Designed ETL data pipeline for integration of real-world EHRs into deep learning module, using Python, SQL, and Apache Spark.

●	Organized and managed close coordination of gRED Artificial Intelligence, data management, and DevOps functions, to bring clinical machine learning from prototype to production.
:::

::: aside

#### PROGRAMMING SKILLS


<i class="fab fa-r-project"></i> `R`

<i class="fab fa-python"></i> `Python`

<i class="fab fa-linux"></i> `Linux`

<i class="fas fa-code-branch"></i> `Git`

<i class="fab fa-github"></i> `GitHub`

<i class="fab fa-markdown"></i> `Markdown`

#### LANGUAGES

**English:** Native

**Spanish:** Native

:::

### Data Curator, Bioinformatics Analyst 

BioMarin Pharmaceutical (Contract)

San Rafael, CA

2018 - 2017 

::: concise
●	Established an integrated genetic and curated literature workflow to predict incidence and prevalence rates of over sixty rare, genetic disorders, with R statistical and visualization packages (ggplot2) and MATLAB.

●	Upgraded and tested RNA-Seq data pipeline (STAR2, SAMtools, Bioconductor) to discern disease contribution of rare variants to neural disease.

●	Spearheaded development of three relational databases derived from Hail HPC population genomics platform, in collaboration with software engineer.

●	Co-authored manuscript submitted to peer-reviewed publication, on prediction of disease severity in metachromatic leukodystrophy.

●	Developed allele-specific association models and GWAS analysis software for two later published manuscripts.
:::

### Research Associate

University of Miami Miller School of Medicine,
The Miami Project to Cure Paralysis, Pearse Lab

Miami, FL

2017 - 2016

::: concise
●	Mentored research associates in developing bioinformatic skills, specifically in multiple sequence alignment and standard molecular biological protocols (restriction enzyme digestion, gel electrophoresis, spectrophotometry).

●	Identified over forty putative conserved vertebrate phosphodiesterase proteins, by using multiple sequence alignment and genomic evidence.
:::

::: aside


Disclaimer {#disclaimer}
--------------------------------------------------------------------------------
<i class="fas fa-download"></i> [Download](https://raw.githubusercontent.com/jsacco1/resume/main/James_Sacco_resume.pdf)


This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown) and [**datadrivencv**](http://nickstrayer.me/datadrivencv/index.html).

Code available on <i class="fa fa-github"></i>  [GitHub](https://github.com/jsacco1/resume). 

Last updated on 2022-01-25.The most recent version of this resume is [available here](https://github.com/jsacco1/resume/blob/main/James_Sacco_resume.pdf).


:::

Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Rowan University, Graduate School of Biomedical Sciences

Master of Biomedical Science (Distinction)

Stratford, NJ

N/A

"Hippocampal long-term potentiation in neurodegenerative disorders"

Prof. Catherine Neary


### Florida International University, The Honors College

BSc, Biological Sciences

Miami, FL

<i class="fas fa-laptop"></i> Professional Development
--------------------------------------------------------------------------------

### Introduction to Genomic Technologies

Johns Hopkins University

Online

2021

### Statistics for Genomic Data Science

Johns Hopkins University

Online

2021

### Software Carpentry for R and UNIX

University of Miami

Miami, FL

2016

::: aside

I am an active member of Biostars [<img src="https://www.biostars.org/static/images/logo6.svg">](https://www.biostars.org/u/40325/) 

I have participated in several bioinformatics and software workshops and Meetup groups, in both Seattle, WA and San Francisco, CA, including: SF/Bay AI Developers Group, San Jose Data Science and AI/ML Meetup, Hacker Dojo, and Seattle Sequencing

#### EXTRA-CURRICULAR

San Francisco chapter of Big Gay Frisbee. 2018 - Present.

Seattle Quake gay rugby union. 2016 - 2017.

:::

Publications {data-icon=file}
--------------------------------------------------------------------------------

### Regulating Axonal Responses to Injury: The Intersection between Signaling Pathways Involved in Axon Myelination and The Inhibition of Axon Regeneration. 

*Frontiers in Molecular Neuroscience.* 2016 Jun 8;9:33.   

N/A

2016

Rao S.N., Pearse D.D. **Manuscript Editor** 

Conference Papers {data-icon=file}
--------------------------------------------------------------------------------

### Prediction of disease severity in metachromatic leukodystrophy using measures of protein activity and a novel phenotype matrix. 

*American Society of Human Genetics Virtual Meeting.* [Poster] **[Forthcoming on bioRxiv.org]**

Online

2020

Trinidad M., Hong X., **Sacco J.**, Nguyen H.P., Clark W.T., Froelich S., LeBowitz J.H., Gelb M.H.

### Association of HGMD and gnomAD variants of unknown significance with prediction of disease incidence and prevalence. 

*BioMarin Internal Research and Development Asilomar Conference.* [Poster]

Pacific Grove, CA

2018

**Sacco J.**, Clark W.T., Yu K., Wu K., LeBowitz J.H. 

<i class="fas fa-laptop"></i> Research Projects
--------------------------------------------------------------------------------

### Characterization of glycan substrates accumulating in GM1 gangliosidosis

*Molecular Genetics and Metabolism Reports.* 2019. 21, p.100524. doi:10.1016/j.ymgmr.2019.100524

N/A

2019

Lawrence R., et al.

### Utilizing ExAC to assess the hidden contribution of variants of unknown significance to Sanfilippo Type B incidence.

*PLoS One.* 2018. 13(7):e0200008. doi:10.1371/journal.pone.0200008

N/A

2018

Clark W.T., et al.

Teaching {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Peer-Led Team Learning (PLTL)

*Florida International University* 

Miami, FL

2009

::: concise
General Biology II facilitator for two concurrent PLTL courses
:::
