---
title: "James Sacco's CV"
author: James Sacco
date: "`r Sys.Date()`"
params:
  pdf_mode:
    value: true
output:
  pagedown::html_resume:
    self_contained: true
---

```{r, include=FALSE}
knitr::opts_chunk$set(
  results='asis', 
  echo = FALSE
)
```

<script src="https://kit.fontawesome.com/1f72d6921a.js" crossorigin="anonymous"></script>

```{css, echo=FALSE}
@import url('https://fonts.googleapis.com/css?family=Lexend+Deca|Montserrat&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Spartan:wght@100;400&display=swap');


/* Give headers playfair font */
h1 {
font-family: 'Spartan', sans-serif;
color: #16617a;
}


:root {
  --sidebar-background-color: #eaf9f5; /* Make sidebar blue */
}


h2{
  font-family: 'Spartan', sans-serif;
 	font-weight: 400;
	font-size: 1.1rem;
	color: #444; 
	text-transform: uppercase;
}


h3{
    font-family: 'Montserrat', sans-serif;
  	font-weight: 500;
  	color: #16617a;

}


a {
	color: #095a95;

}

a:hover {
	text-decoration: underline dotted;
	

}


.aside ul {
	padding-left: 0.2rem;
}

.decorator::after {
	background-color: #23a0ab;
}
```

Aside
================================================================================

```{r}
#add side note where to find online version
if(params$pdf_mode){
  cat("View this CV online with links at https://jsacco1.github.io/cv/")
} else {
 cat("[<i class='fas fa-download'></i> Download the PDF of this CV](https://github.com/jsacco1/jsacco1.github.io/raw/master/cv/James_Sacco_CV.pdf)")
}
```

Contact Info {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i>  jsacco001@gmail.com
- <i class="fas fa-link"></i>  [jsacco1.github.io](https://jsacco1.github.io)
- <i class="fa fa-github"></i>  [github.com](https://github.com/jsacco1)
- <i class="fa fa-linkedin"></i>  [linkedin.com](https://www.linkedin.com/in/james-sacco/)
- <i class="fa fa-phone"></i>  +1 (786)-473-9993

Work authorization: US citizen

For more information, please contact me via email.

Key Skills {#skills}
--------------------------------------------------------------------------------

- **Machine learning** (scikit-learn, Keras)

- **Single cell multi-omics**, **experimental design**, and **robust and reproducible analyses**

- Bioinformatics databases and workflows (Snakemake, Tableau, Docker, Singularity)

- Cloud data analysis (PySpark, Hail), **AWS deep learning stack** (EC2, SageMaker, S3)

- High scientific rigor & willingness to teach and learn about new computational methods and biology


Main
================================================================================

**James Sacco, M.Sc.** {#title}
--------------------------------------------------------------------------------

**Senior Bioinformatician  |  Machine Learning Enthusiast**


--------------------------------------------------------------------------------

Experience {data-icon=briefcase}
--------------------------------------------------------------------------------

### Bioinformatics Programmer III

![UCSF](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/master/images/UCSF_Logo_21_Navy_300dpi_RGB.jpg){width=10%} [Laboratory for Genomics Research](https://www.lgr.bio/), University of California, San Francisco

San Francisco, CA

2024 - 2022

:::concise
● **Conceptualized and designed** a CRISPR-Cas12a library of guide RNAs, with genomic, transcriptomic, and epigenomic data to ensure comprehensive coverage.

● **Investigated drivers** of CRISPR efficacy, using ML, resulting in a 15% increase in targeting accuracy.

● **Led computational analyses** of multiple CRISPR screens in rare diseases, identifying 400 gene hits.

●	Co-authored manuscript on key regulators of human stem cell differentiation.

●	**Collaborated with wet-lab scientists** to support 5 projects, accelerating project timelines by up to 20%.

:::

### Bioinformatics Scientist

![ASC](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/master/images/asctx_logo.jpeg){width=10%} ASC Therapeutics

Milpitas, CA

2022 - 2021

::: concise
● **Conducted comparative analyses** of off-target detection methods in CRISPR screens using single-cell data, employing statistical models to evaluate accuracy.

● Developed a **custom sequence alignment** pipeline for gene editing, improving alignment speed by 80%.

● Automated workflows for sequence alignment, NGS data analysis, and off-target detection, reducing processing time by 60% and enhancing collaboration with experimental teams.

● Implemented company's **first version control** processes, standardizing collaborative software development and data analysis.

:::

### Biomarker Data Analyst II
![GNE](https://seekvectorlogo.com/wp-content/uploads/2018/10/genentech-vector-logo-small.png){width=10%} Genentech (Contract)

South San Francisco, CA

2020 - 2019

::: concise

::: aside

#### PROGRAMMING SKILLS

<i class="fab fa-python"></i> `Python`

<i class="fab fa-r-project"></i> `R`

<i class="fab fa-linux"></i> `Linux`

<i class="fas fa-code-branch"></i> `Git`

![C++](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/e72a5db2c9d73f90a4d5b110bd3fa22bde9ce2eb/images/cpp-svg-44444.svg){width=7%} `C++`

![MATLAB](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/a52737b1c6e96042f1974d3aa700c6df19edcee0/images/matlab-444444.svg){width=7%} `MATLAB`

:::

● **Optimized clinical trials** by streamlining biomarker analysis using ML models, including regression, random forest, and Xgboost, as well as explainable AI.

●	**Improved sample quality** by 10% across 49 clinical trial arms with department's first ML pipeline to study the effects of biomarker operations on the quality of cancer immunotherapy assays.

●	**Upgraded data operations** for 6 data streams, including flow cytometry, biomarker operations, and oncology data warehouses, resulting in more efficient data sharing.
:::

### Data Curator

![GNE](https://seekvectorlogo.com/wp-content/uploads/2018/10/genentech-vector-logo-small.png){width=10%} Genentech (Contract)

South San Francisco, CA

2019 - 2018 

::: concise
●	Integrated RNA assays, enhancing **AI/ML module performance** efficacy by at least 10%.

●	**Enabled AI/ML team** in patient stratification with robust data pipelines for static and time series data points.

●	Coordinated data governance and integration between AI/ML, data management, and DevOps functions, to bring **clinical ML** from prototype to production.
:::

### Data Curator, Bioinformatics Analyst 

![BioMarin](https://www.wmprocess.com/wp-content/uploads/2020/07/biomarin.png){width=10%} BioMarin Pharmaceutical (Contract)

San Rafael, CA

2018 - 2017 

::: concise
●	**Published author** on ARSA gene research for metachromatic leukodystrophy, adding to rare disease drug development.

●	**Streamlined data analysis** and visualization using R, Python, and MATLAB, improving data analysis speed by 70%.

●	**Developed workflows** for pathogenic variant identification across 60 rare disorders, resulting in more accurate prediction of disease prevalence and incidence.

●	**Prototyped relational databases** with Hail for efficient genomic data storage, improving data management and accessibility.

:::

:::aside

Active member of Biostars [<img src="https://www.biostars.org/static/images/logo6.svg">](https://www.biostars.org/u/40325/) 

I have participated in bioinformatics and software workshops and *Meetup.com* groups, in both Seattle, WA and San Francisco, CA, including: 

*SF/Bay AI Developers Group*

*San Jose Data Science and AI/ML Meetup*

*Hacker Dojo*

*Seattle Sequencing*

#### LANGUAGES

**English:** Native Proficiency

**Spanish:** Native Proficiency

**Dutch:** Intermediate Proficiency

:::

### Research Associate

![UM](https://1000logos.net/wp-content/uploads/2017/07/University-of-Miami-Logo.png){width=10%} Pearse Lab: The Miami Project to Cure Paralysis, The Miller School of Medicine, University of Miami

Miami, FL

2017 - 2016

::: concise

●	Identified over forty putative **conserved vertebrate phosphodiesterase proteins**, using multiple sequence alignment and genomic evidence.
:::

Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Rowan University, Graduate School of Biomedical Sciences

Master of Biomedical Science

Stratford, NJ

N/A

### Florida International University, The Honors College

BSc, Biological Sciences

Miami, FL

<i class="fas fa-laptop"></i> Professional Development
--------------------------------------------------------------------------------

### [Python for Data Science: Fundamentals Part I Course](http://links.iterable.com/u/click?_t=4a16705ff7784260b1b139ad7ac93ebf&_m=34ed48a7273845da8e2eebf7f103638a&_e=A8KtzNG8hQmSIiZw8a6IRZSe870W1l30iT3FAuPZkLT4brzWYfnDAAktVxaO27-kZSk2JLhvIAXpJ4YxVC9PYPu-CGfFa6kbwEUHOvXHyHZBvZ6zW5Kpg2rsCwx-xdk9DoKhhPMaGihG_0rexFh258pNbM0x2TtSTF3esfilJpopBdTrEPPcFQwb6nd6GPseq8ilYPk7Mm4pOxPyR6sH1kjz-Jn91VF1iHy52kN8-Zg1z22hANXI-hk7BxqpYBE-n5_NkFxST4dQ2SyxlwLL-cwBG6bd1CTpxuYYOiw-eJY%3D)

Dataquest.io

Online

2022

### [Introduction to Genomic Technologies](https://coursera.org/share/349b7ec7b038591b3ea316ef9d57f91c)

Johns Hopkins University

Online

2021

### [Statistics for Genomic Data Science](https://coursera.org/share/0fbdae3d36a310f4653b95ca375d0009)

Johns Hopkins University

Online

2021

### [Software Carpentry for R and UNIX](https://mkcor.github.io/2016-03-14-university-of-miami/)

University of Miami

Miami, FL

2016


Publications {data-icon=file}
--------------------------------------------------------------------------------

### [Predicting disease severity in metachromatic leukodystrophy using protein activity and a patient phenotype matrix.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03001-z) 

*Genome Biology.* 2023 Jul 24:172.   

N/A

2023

Trinidad M., Hong X., Froelich S., Daiker J., **Sacco J.**, Phuc Nguyen H., Campagna M., Suhr D., Suhr T., LeBowitz J.H., Gelb M.H., Clark W.T.

### [Regulating Axonal Responses to Injury: The Intersection between Signaling Pathways Involved in Axon Myelination and The Inhibition of Axon Regeneration.](https://www.frontiersin.org/articles/10.3389/fnmol.2016.00033/full) 

*Frontiers in Molecular Neuroscience.* 2016 Jun 8;9:33.   

N/A

2016

Rao S.N., Pearse D.D. **Manuscript Editor** 

Conferences {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------
### [Single Cell Analyses](https://meetings.cshl.edu/meetings.aspx?meet=SINGLE&year=23) 

![Cold Spring Harbor Laboratory Logo](https://meetings.cshl.edu/images/cshl.png){width=10%} *Cold Spring Harbor Laboratory Single Cell Analyses Meeting.*

Cold Spring Harbor, New York

2023


### [Prediction of disease severity in metachromatic leukodystrophy using measures of protein activity and a novel phenotype matrix.](https://www.abstractsonline.com/pp8/#!/9070/presentation/2893) 

![American Society of Human Genetics](https://ctsi.mcw.edu/images/ashg-logo-713x401.png){width=10%} *American Society of Human Genetics Virtual Meeting.* [Poster]

Online

2020

Trinidad M., Hong X., **Sacco J.**, Nguyen H.P., Clark W.T., Froelich S., LeBowitz J.H., Gelb M.H.

### Association of HGMD and gnomAD variants of unknown significance with prediction of disease incidence and prevalence. 

![BioMarin](https://www.wmprocess.com/wp-content/uploads/2020/07/biomarin.png){width=10%} *BioMarin Internal Research and Development Asilomar Conference.* [Poster]

Pacific Grove, CA

2018

**Sacco J.**, Clark W.T., Yu K., Wu K., LeBowitz J.H. 

<i class="fas fa-laptop"></i> Research Projects
--------------------------------------------------------------------------------

### [Characterization of glycan substrates accumulating in GM1 gangliosidosis](https://pubmed.ncbi.nlm.nih.gov/31720227/)

*Molecular Genetics and Metabolism Reports.* 2019. 21, p.100524.

N/A

2019

Lawrence R., et al.

### [Utilizing ExAC to assess the hidden contribution of variants of unknown significance to Sanfilippo Type B incidence](https://pubmed.ncbi.nlm.nih.gov/29979746/)

*PLoS One.* 2018. 13(7):e0200008.

N/A

2018

Clark W.T., et al.

### [Phosphodiesterase Inhibitors as a Therapeutic Approach to Neuroprotection and Repair](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5412282/)

*International Journal of Molecular Sciences.* 2017. 18(4): 696.

N/A

2017

::: aside

#### HOBBIES

Stonewall Sports kickball. San Francisco, CA. 2023.

Ultimate frisbee player. San Francisco, CA. 2018 -- Present.

Rugby at *Seattle Quake RFC*. Seattle, WA. 2017.

Literature

Classical Music

Language Learning

:::

Knott E.P., et al.

Oral Communications {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### [Data Visualization of RNA-Seq Differential Expression Analysis with DESeq2](https://github.com/jsacco1/rnaseq/blob/main/Standard%20RNA-Seq%20Plots%20and%20Graphs.pptx)

![HHMI](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/master/images/logo.png){width=10%} [Doudna Lab](https://doudnalab.org/): Howard Hughes Medical Institute, Department of Molecular and Cell Biology, University of California, Berkeley

Berkeley, CA

2022

:::concise
Topics: Introduction to RNA-Seq, Standard Workflows with DESeq2, Reference-Based Assembly, Exploratory Data Analysis, Statistical Analysis, Data Visualization, Gene Set Enrichment, Variant Annotation

●	Presented publication-quality RNA-Seq data summaries and representations to post-doctoral associates 
:::

### [Changes in chromatin accessibility are not concordant with transcriptional changes for single-factor perturbations](https://www.biorxiv.org/content/10.1101/2022.02.03.478981v1)


![UCSF](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/master/images/UCSF_Logo_21_Navy_300dpi_RGB.jpg){width=10%} [Laboratory for Genomics Research](https://www.lgr.bio/) & Department Biochemistry & Biophysics, University of California, San Francisco

San Francisco, CA

2022

:::concise
●	Presented now-published preprint in contrast with established research at internal Journal Club to inform future experimental design in epigenetic editing.
:::

Teaching {data-icon=book}
--------------------------------------------------------------------------------

### [Peer-Led Team Learning](https://loadpltl.fiu.edu/)

*Florida International University* 

Miami, FL

2009

::: concise
General Biology II facilitator for two concurrent PLTL courses
:::

::: aside

Disclaimer {#disclaimer}
--------------------------------------------------------------------------------

Made with 💘 using R [**pagedown**](https://github.com/rstudio/pagedown).
 
Last update: `r Sys.Date()`.The most recent version of this resume is [available here](https://github.com/jsacco1/jsacco1.github.io/blob/master/cv/index.pdf).
:::
