---
title: "James Sacco's CV"
author: James Sacco
date: "`r Sys.Date()`"
params:
  pdf_mode:
    value: true
output:
  pagedown::html_resume:
    self_contained: true
---

```{r, include=FALSE}
knitr::opts_chunk$set(
  results='asis', 
  echo = FALSE
)
```

<script src="https://kit.fontawesome.com/1f72d6921a.js" crossorigin="anonymous"></script>

```{css, echo=FALSE}
@import url('https://fonts.googleapis.com/css?family=Lexend+Deca|Montserrat&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Spartan:wght@100;400&display=swap');


/* Give headers playfair font */
h1 {
font-family: 'Spartan', sans-serif;
color: #16617a;
}


:root {
  --sidebar-background-color: #eaf9f5; /* Make sidebar blue */
}


h2{
  font-family: 'Spartan', sans-serif;
 	font-weight: 400;
	font-size: 1.1rem;
	color: #444; 
	text-transform: uppercase;
}


h3{
    font-family: 'Montserrat', sans-serif;
  	font-weight: 500;
  	color: #16617a;

}


a {
	color: #095a95;

}

a:hover {
	text-decoration: underline dotted;
	

}


.aside ul {
	padding-left: 0.2rem;
}

.decorator::after {
	background-color: #23a0ab;
}
```

Aside
================================================================================

```{r}
#add side note where to find online version
if(params$pdf_mode){
  cat("View this CV online with links at https://jsacco1.github.io/cv/")
} else {
 cat("[<i class='fas fa-download'></i> Download the PDF of this CV](https://github.com/jsacco1/jsacco1.github.io/raw/master/cv/James_Sacco_CV.pdf)")
}
```

Contact Info {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i> jsacco001@gmail.com
- <i class="fas fa-link"></i> [jsacco1.github.io](https://jsacco1.github.io)
- <i class="fa fa-github"></i> [github.com](https://github.com/jsacco1)
- <i class="fa fa-linkedin"></i> [linkedin.com](https://www.linkedin.com/in/james-sacco/)
- <i class="fa fa-phone"></i> +1 (786)-473-9993

For more information, please contact me via email

Key Skills {#skills}
--------------------------------------------------------------------------------

- Expert programming in **Python**, **R**, and **Linux shell scripting**

- Extensive experience with **statistical, NGS data analyses** and public genomic repositories

- Proficiency with _in silico_ **CRISPR gene editing** modules

- Machine learning and applied statistics

- Data engineering & pipeline development (AWS, Airflow, PySpark)

- High scientific rigor and eagerness to teach and learn about new computational methods and biology


Main
================================================================================

**James Sacco** {#title}
--------------------------------------------------------------------------------

**Bioinformatics Scientist**  
**Department of Gene and Cell Therapy**  
**ASC Therapeutics**  
**Milpitas, CA 95035**

Self-motivated **bioinformatics programming professional** with **six years** of experience in support of **rare disease and cancer therapies**. Areas of expertise include **transcriptomics**, **CRISPR gene editing**, development of robust, reproducible **data workflows** for scientific user interfaces. 

An analytical thinker and quick learner, with a broad understanding of **statistical and machine learning methods**. Published author of research studies in the integration of genotype and phenotype data to **model disease progression and epidemiology**.

--------------------------------------------------------------------------------

Selected Experience {data-icon=briefcase}
--------------------------------------------------------------------------------

### Bioinformatics Scientist

![ASC](https://medicalnewsupdates.com/wp-content/uploads/2021/07/asc_therapeutics_logo_color_print-01_square.jpg){width=10%} ASC Therapeutics (Contract)

Milpitas, CA

Present - 2021

::: concise
‚óè Quantify and characterize **CRISPR gene modifications**, with both custom and open-source tools for genomic data analysis.

‚óè Query, retrieve, and integrate data from public genomics databases, to enhance on-target gene editing with sequence alignment protocols. 

‚óè Examine concordance and sensitivity of five open-source CRISPR off-target detection methods (**CALITAS**, **CHANGE-Seq**, **CRISPResso**, **GUIDE-Seq**, and **Cas-OFFinder**).

‚óè Streamline high-performance analyses for bench scientists by constructing **custom data pipelines**, **command line tools**, and **user interfaces**.

‚óè Generate reproducible reports for publication-quality manuscripts.
:::

### Biomarker Data Analyst II
![GNE](https://seekvectorlogo.com/wp-content/uploads/2018/10/genentech-vector-logo-small.png){width=10%} Genentech (Contract)

South San Francisco, CA

2020 - 2019

::: concise
‚óè Established department-first **machine learning pipeline** to study effects of biomarker operations on quality of cancer immunotherapy assays.

‚óè	Upgraded **data operations** for six data streams, including flow cytometry, biomarker operations, and oncology data warehouses.

‚óè	Monitored and solved sample data quality issues for two cancer immunotherapy trial arms.
:::

### Data Curator

![GNE](https://seekvectorlogo.com/wp-content/uploads/2018/10/genentech-vector-logo-small.png){width=10%} Genentech (Contract)

South San Francisco, CA

2019 - 2018 

::: aside

#### PROGRAMMING SKILLS

<i class="fab fa-python"></i> `Python` ‚òÖ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fab fa-r-project"></i> `R` ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fab fa-linux"></i> `Linux` ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fas fa-code-branch"></i> `Git` ‚òÖ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fab fa-github"></i> `GitHub` ‚òÖ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fab fa-markdown"></i> `Markdown`‚òÖ‚òÖ‚òÖ‚òÖ‚òÖ

<i class="fas fa-database"></i> `SQL` ‚òÖ‚òÖ‚òÖ

<i class="far fa-chart-bar"></i> `Dash` ‚òÖ‚òÖ‚òÖ

![C++](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/e72a5db2c9d73f90a4d5b110bd3fa22bde9ce2eb/images/cpp-svg-44444.svg){width=7%} `C++` ‚òÖ‚òÖ‚òÖ

![MATLAB](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/a52737b1c6e96042f1974d3aa700c6df19edcee0/images/matlab-444444.svg){width=7%} `MATLAB` ‚òÖ‚òÖ‚òÖ

:::

::: concise
‚óè	Collaborated with AI engineers to create an ETL pipeline for **multi-modal survival prediction and patient stratification**, by using Python ML and R to integrate **gene panel** and **RNA-Seq data**.

‚óè	Designed ETL data pipeline for integration of **real-world EHRs** into **deep learning** module, using Python, SQL, and Apache Spark.

‚óè	Organized and managed close coordination of Artificial Intelligence, data management, and DevOps functions, to bring **clinical machine learning** from prototype to production.
:::

### Data Curator, Bioinformatics Analyst 

![BioMarin](https://www.wmprocess.com/wp-content/uploads/2020/07/biomarin.png){width=10%} BioMarin Pharmaceutical (Contract)

San Rafael, CA

2018 - 2017 

::: concise
‚óè	Established an integrated genetic and curated literature workflow to predict incidence and prevalence rates of **over sixty rare, genetic disorders,** with **statistical and data visualization R packages (ggplot2)** and **MATLAB**.

‚óè	Upgraded and tested **RNA-Seq data pipeline (STAR2, SAMtools, Bioconductor)** to discern disease contribution of **rare variants** to neural disease.

‚óè	Spearheaded development of three relational databases derived from **Hail population genomics platform**, in collaboration with software engineer.

‚óè	Co-authored manuscript submitted to peer-reviewed publication, on **prediction of disease severity** in metachromatic leukodystrophy.

‚óè	Developed **allele-specific association models and GWAS analysis software** for two later published manuscripts.
:::

### Research Associate

![UM](https://1000logos.net/wp-content/uploads/2017/07/University-of-Miami-Logo.png){width=10%} Pearse Lab: The Miami Project to Cure Paralysis, The Miller School of Medicine, University of Miami

Miami, FL

2017 - 2016

::: concise
‚óè	Mentored research associates in developing bioinformatic skills, specifically in **multiple sequence alignment** and **standard molecular biological protocols** (restriction enzyme digestion, gel electrophoresis, spectrophotometry).

‚óè	Identified over forty putative **conserved vertebrate phosphodiesterase proteins**, by using multiple sequence alignment and genomic evidence.
:::

Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Rowan University, Graduate School of Biomedical Sciences

Master of Biomedical Science (Distinction)

Stratford, NJ

N/A

"Hippocampal long-term potentiation in neurodegenerative disorders"

Related coursework:

‚óè	Biochemistry

‚óè	Molecular Biology

‚óè	Cell Biology

‚óè	Microbiology

‚óè	Pharmacology

### Florida International University, The Honors College

BSc, Biological Sciences

Miami, FL

<i class="fas fa-laptop"></i> Professional Development
--------------------------------------------------------------------------------

:::aside

Active member of Biostars [<img src="https://www.biostars.org/static/images/logo6.svg">](https://www.biostars.org/u/40325/) 

I have participated in bioinformatics and software workshops and *Meetup.com* groups, in both Seattle, WA and San Francisco, CA, including: 

*SF/Bay AI Developers Group*

*San Jose Data Science and AI/ML Meetup*

*Hacker Dojo*

*Seattle Sequencing*

#### LANGUAGES

**English:** Native

**Spanish:** Bilingual Proficiency

:::

### [Bulk and Single Cell Transcriptomics Training Program](https://learn.omicslogic.com/programs/bulk-and-single-cell-transcriptomics-training)

Pine Biotech, Tauber Bioinformatics Research Center

Online

2022

### [Python for Data Science: Fundamentals Part I Course](http://links.iterable.com/u/click?_t=4a16705ff7784260b1b139ad7ac93ebf&_m=34ed48a7273845da8e2eebf7f103638a&_e=A8KtzNG8hQmSIiZw8a6IRZSe870W1l30iT3FAuPZkLT4brzWYfnDAAktVxaO27-kZSk2JLhvIAXpJ4YxVC9PYPu-CGfFa6kbwEUHOvXHyHZBvZ6zW5Kpg2rsCwx-xdk9DoKhhPMaGihG_0rexFh258pNbM0x2TtSTF3esfilJpopBdTrEPPcFQwb6nd6GPseq8ilYPk7Mm4pOxPyR6sH1kjz-Jn91VF1iHy52kN8-Zg1z22hANXI-hk7BxqpYBE-n5_NkFxST4dQ2SyxlwLL-cwBG6bd1CTpxuYYOiw-eJY%3D)

Dataquest.io

Online

2022

### [Introduction to Genomic Technologies](https://coursera.org/share/349b7ec7b038591b3ea316ef9d57f91c)

Johns Hopkins University

Online

2021

### [Statistics for Genomic Data Science](https://coursera.org/share/0fbdae3d36a310f4653b95ca375d0009)

Johns Hopkins University

Online

2021

### [Software Carpentry for R and UNIX](https://mkcor.github.io/2016-03-14-university-of-miami/)

University of Miami

Miami, FL

2016


Publications {data-icon=file}
--------------------------------------------------------------------------------

### [Regulating Axonal Responses to Injury: The Intersection between Signaling Pathways Involved in Axon Myelination and The Inhibition of Axon Regeneration.](https://www.frontiersin.org/articles/10.3389/fnmol.2016.00033/full) 

*Frontiers in Molecular Neuroscience.* 2016 Jun 8;9:33.   

N/A

2016

Rao S.N., Pearse D.D. **Manuscript Editor** 

Conference Papers {data-icon=file}
--------------------------------------------------------------------------------

### [Prediction of disease severity in metachromatic leukodystrophy using measures of protein activity and a novel phenotype matrix.](https://www.abstractsonline.com/pp8/#!/9070/presentation/2893) 

![American Society of Human Genetics](https://ctsi.mcw.edu/images/ashg-logo-713x401.png){width=10%} *American Society of Human Genetics Virtual Meeting.* [Poster] **[Forthcoming on bioRxiv.org]**

Online

2020

Trinidad M., Hong X., **Sacco J.**, Nguyen H.P., Clark W.T., Froelich S., LeBowitz J.H., Gelb M.H.

### Association of HGMD and gnomAD variants of unknown significance with prediction of disease incidence and prevalence. 

![BioMarin](https://www.wmprocess.com/wp-content/uploads/2020/07/biomarin.png){width=10%} *BioMarin Internal Research and Development Asilomar Conference.* [Poster]

Pacific Grove, CA

2018

::: aside

Avid supporter of the San Francisco Symphony and San Francisco Conservatory of Music

Ultimate frisbee at *Big Gay Frisbee.* San Francisco, CA. 2018 -- Present.

Salsa dance at *In Lak‚Äôech Dance Academy.* Oakland, CA. 2019.

Rugby at *Seattle Quake RFC*. Seattle, WA. 2016 -- 2017.

:::

**Sacco J.**, Clark W.T., Yu K., Wu K., LeBowitz J.H. 

<i class="fas fa-laptop"></i> Research Projects
--------------------------------------------------------------------------------

### [Characterization of glycan substrates accumulating in GM1 gangliosidosis](https://pubmed.ncbi.nlm.nih.gov/31720227/)

*Molecular Genetics and Metabolism Reports.* 2019. 21, p.100524.

N/A

2019

Lawrence R., et al.

### [Utilizing ExAC to assess the hidden contribution of variants of unknown significance to Sanfilippo Type B incidence](https://pubmed.ncbi.nlm.nih.gov/29979746/)

*PLoS One.* 2018. 13(7):e0200008.

N/A

2018

Clark W.T., et al.

### [Phosphodiesterase Inhibitors as a Therapeutic Approach to Neuroprotection and Repair](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5412282/)

*International Journal of Molecular Sciences.* 2017. 18(4): 696.

N/A

2017

Knott E.P., et al.

Oral Communications {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### [Data Visualization of RNA-Seq Differential Expression Analysis with DESeq2](https://github.com/jsacco1/rnaseq/blob/main/Standard%20RNA-Seq%20Plots%20and%20Graphs.pptx)

![HHMI](https://raw.githubusercontent.com/jsacco1/jsacco1.github.io/master/images/logo.png){width=10%} [Doudna Lab](https://doudnalab.org/): Howard Hughes Medical Institute, Department of Molecular and Cell Biology, University of California, Berkeley

Berkeley, CA

2022

:::concise
Topics: Introduction to RNA-Seq, Standard Workflows with DESeq2, Reference-Based Assembly, Exploratory Data Analysis, Statistical Analysis, Data Visualization, Gene Set Enrichment, Variant Annotation

‚óè	Presented publication-quality RNA-Seq data summaries and representations to post-doctoral associates 
:::

Teaching {data-icon=book}
--------------------------------------------------------------------------------

### [Peer-Led Team Learning](https://loadpltl.fiu.edu/)

*Florida International University* 

Miami, FL

2009

::: concise
General Biology II facilitator for two concurrent PLTL courses
:::

::: aside

Disclaimer {#disclaimer}
--------------------------------------------------------------------------------

Made with üíò using R [**pagedown**](https://github.com/rstudio/pagedown).
 
Last update: `r Sys.Date()`.The most recent version of this resume is [available here](https://github.com/jsacco1/jsacco1.github.io/blob/master/cv/index.pdf).
:::